### ISSUE TITLE
Use case: As a supporter, I'd like to pay in my own currency

### ISSUE THREAD

**Issue by arnaudgo** [online-go/online-go.com#366]
### Summary

**- edit -**

**business case/use case:** As a supporter, I'd like to pay in my own currency

### Environment

any

### Steps To Reproduce:

ask if needed

### Additional Information:

any

Arnaud

**Comment by SoftwUser** [comment]
I don't understand that one.
Paying with credit card or Paypal, you will be charged in your local currency.
So what's the issue?

Or do you insist on seeing the reference amount in for example Nakfa instead of Dollar?

**Comment by arnaudgo** [comment]
Hello @SoftwUser :-) 

You're right, the (Github) issue is not very clear since at the final moment, my bank account is debited in my own currency anyway.

The first and minor issue is a comestic one: it saves the user of having to make an approximate conversion from usd to his currency. A courtesy would be just to display both amounts: one in usd and on in his currency.

The second issue is that one who makes foreign transactions might be charged supplement fees, depending on what payment card is used, etc. Some may be charged by the merchant, so it's maybe easy to display the exact amount that will debited on the customer account. Some may be charged by the bank.

Back to OGS support topic, when one gives some money to OGS, I imagine it might be possible that some banks charge some currency change fees.

I'm not one of those, I've always been charged in USD so far. And I also have little knowledge of the topic actually. So, to anyone, feel free to correct me if I'm wrong.
Still, I imagine that some users might have that in mind.

Or what do you think ?

Arnaud

**Comment by SoftwUser** [comment]
Thank you, @arnaudgo for elaborating a bit. Now I see a bit better, what your motivation is.

So let me answer to your issues:

Indeed - before donating - it is up to the user here to convert the Dollar to his local currency. What is the problem? If you really have no idea about the relation, let `google` do it for you. Enter `5 usd` and you get the result. What would be a "courtesy" on the one hand would be an enormous effort on the other hand: There are more than 180 currencies worldwide. And the exchange rates of most of them permanently change. The "service" you suggest would be by far beyond the means for our online-go platform, IMHO.

Correct, financial services charge a fee. But this is by far smaller than handling 180 different currencies. The only way to avoid fees from a donator's perspective would be to send cash to @anoek , but don't do this to him: He would need to exchange cash to his local currency, which would be way more expensive. Apart from the complicated handling of course. 

Also:  In case your suggestion would be realized  - at any point currencies must be converted by a financial service, if not at the donator's side - then on the receiver's side. In this case the fees would be in absurd relations compared to the small donation amounts.

On the one hand you are right, there is a bit of converting calculation and there are some fees. But on the other hand there is no cheaper and more effective way of collecting donations in an international/global perspective than the way it is offered on this platform, at least none that I am aware of.

**Comment by arnaudgo** [comment]
Hi,

On the first point:

There's no problem, that's why I talked about a cosmetic issue. Let me do a comparison to make it clearer: it might be convenient to send to the web user pages with font "medium"-size. In my humble opinion, it's a bit cumbersome to display in small font, argumenting there's no problems with that since the user can do Ctrl-+ on its browser to get it bigger.
On my point of view, it's not a good philosophy. And as a end user, I'm not the one who can say if it's worth or not. And, it's common sense not to provide us this enhancement if it's overkill. 

It's up to us to judge if fees are in absurd proportion or not, It's a pity you downsize it. Anyway... given that fees might exist, I'm thinking of those who want to anticipate how much the donation will really represent, no matter if you, or whoever, think the difference will be big or not.

Again, I'm just expressing an idea and I don't have questions, thanks :)

Arnaud

**Comment by SoftwUser** [comment]
`And I also have little knowledge of the topic actually. So, to anyone, feel free to correct me if I'm wrong.`

To be honest, I actually did not intend to correct you.

The only thing I wanted to do is to point out the economic implications and consequences - based on more than 30 years of professional experience on banking and payment transactions.
So I did not downsize anything - I just presented facts,

Facts that might be considered or discarded when approving or denying your "idea" that you expressed with "As a supporter, I'd like to pay in my own currency".
